checkov/main.py

```diff
@@ -445,6 +445,8 @@ def run(self, banner: str = checkov_banner, tool: str = checkov_tool, source_typ
                           '(but note that this will not include any custom platform configurations or policy metadata).',
                           file=sys.stderr)
                     self.exit_run()
+
+            bc_integration.get_runtime_run_config()
```

Comment:
can we do it in parallel to `get_platform_run_config`?

Reply:
I want to be consistent with all other platform calls.